homeassistant/components/ozw/strings.json

```diff
@@ -1,11 +1,21 @@
 {
   "config": {
     "step": {
-      "user": {
-        "title": "Confirm set up"
+      "on_supervisor": {
+        "title": "Select connection method",
```

Comment:
Only in advanced mode should we allow the user to set up the OZW controller themselves.

Reply:
This is a follow up. I've left it out on purpose.